bin/scrapeReference42021.js

```diff
@@ -90,28 +98,42 @@ function parseCriteria(criterias, topicNumber) {
  * @returns {string} String
  */
 function getWcagLevel(wcag) {
-	return wcag
-		.split(' ')
-		.pop()
-		.replace(/[{()}]/g, '');
+	const matches = wcag.match(/\((A+)\)$/);
+	return matches ? matches[1] : '';
 }
 
 /**
  * @param {object} tests
  * @param {number} criteriumId
  * @param {string} topicNumber
- * @return {Array<object>} Array[object] 
+ * @return {Array<object>} Array[object]
  */
 function getTests(tests, criteriumId, topicNumber) {
-	let testsElements = [];
-	for (const key in tests) {
-		let testsNumbering = `${topicNumber}.${criteriumId}.${key}`;
-		if (Object.hasOwnProperty.call(tests, key)) {
-			testsElements = [
-				...testsElements,
-				{id: testsNumbering, title: marked(objectToList(tests[key]), { renderer })}
-			];
+	return Object.entries(tests).map(([key, _]) => ({
```

Comment:
Si tu itères que sur les clefs de ton objet tu peux utiliser directement `Object.keys`.